man/zeek-client.1

```diff
@@ -0,0 +1,166 @@
+.TH ZEEK\-CLIENT "1" "2023\-09\-11" "zeek\-client" "Generated Python Manual"
```

Comment:
The `argparse_manpage` module by defaults embeds the time the build job was run into the generated manpage. This probably makes sense, but interferes with running this from a pre-commit hook as the output is not reproducible.

The module supports reproducible builds with [SOURCE_DATE_EPOCH](https://reproducible-builds.org/docs/source-date-epoch/). We could set this in `build.py` before calling `Manpage`, e.g.,

```python
os.environ['SOURCE_DATE_EPOCH'] = "1694507965"
```

This is unfortunately pretty opaque and would need a comment.

Git does not preserve mtime, so e.g., the following unfortunately would not work:

```python
os.environ['SOURCE_DATE_EPOCH'] = f"{int(os.path.getmtime(__file__))}"
```

If you believe this is too much hassle to implement, we should not use this as a pre-commit hook.

Reply:
Thanks! I just removed the date altogether.